src/grisbi_osx.m

```diff
@@ -58,20 +58,29 @@
     return program_name;
 }
 
-static char const *get_program_dir()
+static gchar *get_program_dir()
```

Comment:
I like to have `get_program_dir(void)` when the function takes no parameter.

Reply:
désolé c'est poussif, j'aurai du relire mon code :)